src/server/cmd/pachd/main.go

```diff
@@ -238,6 +240,11 @@ func doSidecarMode(appEnvObj interface{}) error {
 				eprsclient.RegisterAPIServer(s, enterpriseAPIServer)
 
 				healthclient.RegisterHealthServer(s, health.NewHealthServer())
+				debugclient.RegisterDebugServer(s, debugserver.NewDebugServer(
+					"", // no name for pachd servers
```

Comment:
It might be nice to differentiate btw the pachd pod vs worker sidecars. And if we could populate the pod name for pachd sidecars that may be helpful too. Unless I'm missing something that you'd get from the debug output that would ID the pod/container source? 

I could easily see a case where you have many pipelines, and you want to see which container/pod a pachd sidecar issue happened on. Without this info, you wouldn't even know which pipeline's sidecar had the issue (much less which worker in particular if parallelism is used). Most of the time we see pachd sidecar issues it has something to do w the usage pattern from a pipeline, so this seems helpful. That said ... I'm not sure you'll have the `appEnv.PodName` as you do in the worker. But I think we can string that through.

Reply:
Reading more of the code ... it looks like you have some support for dumping by pipelines (via the `pipelineRcName` param to the `Clients()` method), but it doesn't seem to used by the `Dump()` method at all.

In any case, I think it'd be helpful to provide the pod name. Even within a pipeline knowing which pod is running which goro's would be helpful. I think it's likely we'd want to know which datums the worker was processing to see if we could reproduce the issue. By knowing the worker/podname you could get the logs to get the datum list.

Reply:
That should be pretty doable. Right now we're not actually pulling debug dumps from the side cars, we can do it but it didn't seem super important to me since the main thing that will tell you is if there's a hanging call to the API but the only thing that makes calls to it is the worker and so we'd see the client side of the hanging call.

Reply:
Fair enough. If we're not adding pachd sidecars, this is less helpful.